.github/workflows/ci.yml

```diff
@@ -13,8 +13,8 @@ jobs:
         working-directory: "nx"
     strategy:
       matrix:
-        elixir: [1.11.4]
-        otp: [23.3.4]
+        elixir: ["1.12.0-rc.1"]
```

Comment:
@josevalim We can pass a list of versions here and to `:otp` and Github Actions will run tests using the cartesian product of the versions. Might be handy later on